k8s/migration/pkg/utils/credutils.go

```diff
@@ -293,14 +293,28 @@ func GetOpenstackInfo(ctx context.Context, k3sclient client.Client, openstackcre
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to extract all networks")
 	}
-
+	allPagesv, err := schedulerstats.List(openstackClients.BlockStorageClient, nil).AllPages()
```

Comment:
How about naming this `allStoragePoolPages` sticking to similar pattern as rest of the code.